### PR TITLE
Clarify cancelability during scroll

### LINKED ISSUE #76: Adding the fling intervention behavior to TouchEvent's spec
Under '5.4 Cancelability of touch events', I want to add a new paragraph.

'While there is an active fling animation, all the touch events generated from a user agent on the active flinging layer may be uncancelable.'

Please free feel to comment on this.

Below is the link to the intent-to-ship draft,
https://docs.google.com/document/d/16D5Hle3-zG_kXjXJbQALD2OWOLmsMkGz1r1BthE2V7Q/edit 

Thank you!

### PR DESCRIPTION
**Pull request by RByers**
Fixes #76

### CODE REVIEW

**Comment by RByers** [comment]
@patrickhlauke @staktrace @patrickkettner WDYT?  As I understand it this is close to what Gecko and Edge do, except that you may not fire the touch events at ALL during the scroll, right?  Safari fires uncancelable ones during a scroll, and an uncancelable `touchstart` on fling but still a cancelable `touchmove`.

See some more discussion [here](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/wJGmsdYxwrg)

**Comment by staktrace** [comment]
Yeah, in Gecko we don't fire the touch events at all. But putting this in the spec seems like a good clarification.

I think the "may not" wording could be improved though - usually "may not" means that it shouldn't happen (e.g. "you may not go the washroom") but in this case what you're trying to say is that the browser might "not suspend the motion of an element". I suggest rewording to either:
... User Agents may continue the motion ...
... User Agents might not suspend the motion ...
(or something else).

**Comment by patrickhlauke** [comment]
Agree with @staktrace that "may not" has a different connotation to what we're trying to say (even at the start of the sentence, which is not modified by this PR). it's difficult to come up with an unambiguous and short way to say "the UA may choose not to do X", so for clarity I'd suggest turning those statements around into the opposite, to say what the UA may actively do (rather than what the UA may choose not to do), if that makes sense...

File: index.html [hunk 1]
```diff
@@ -734,7 +734,10 @@ <h3 id="cancelability">Cancelability of touch events</h3>
           <a href="#dfn-canceled-event">Canceling</a> a touch event can prevent or otherwise
           interrupt scrolling (which could be happening in parallel with script execution).
           For maximum scroll performance, a user agent may not wait for each touch event
-          associated with the scroll to be processed to see if it will be canceled.  In such cases
+          associated with the scroll to be processed to see if it will be canceled. In
+          particular, User Agents may not suspend the motion of an element which is currently
```
Comment (patrickhlauke) [inline index.html]: Minor editorial thing: we have been using "user agent"/"user agents" (lowercase) throughout

**Comment by patrickhlauke** [comment]
Possibly something like

> However, for maximum scroll performance, a user agent may decide to process touch events associated with the scroll, without waiting to see if it will be canceled. In particular, user agents may continue the motion of an element which is currently moving due to a touch scroll, either as a result of an earlier event in the touch sequence, or due to momentum left over from a previous touch sequence, rather than suspending the motion.

**Comment by patrickhlauke** [comment]
Wondering if the second sentence can be condensed a bit - unless I'm misunderstanding the overall meaning, I'm not quite clear why we're now talking about motion of elements, when (i think) what we want to say in short is:

> In particular, regardless of any canceled touch events, a user agents may continue an ongoing scroll motion – caused by an earlier event in the touch sequence, or by momentum left over from a previous touch sequence (e.g. a "fling" gesture) – rather than halting the scrolling.

**Comment by patrickhlauke** [comment]
@RByers is this still relevant/needed?

**Comment by patrickhlauke** [comment]
@RByers ping?

**Comment by RByers** [comment]
Oh, sorry that I apparently just dropped this. It's still relevant in that AFAIK it describes behaviour in multiple browsers. But it's not necessarily "needed", just a small clarification. If you think it ads value to describe this case explicitly then I'm fine landing the text your proposed. Otherwise I'm fine just leaving it all as is.

**Comment by patrickhlauke** [comment]
@RByers @staktrace coming back very late, but: could you cast your eye one last time over this PR which now uses the rewording I proposed above? If all good, I'll merge.

**Comment by RByers** [comment]
Thanks @patrickhlauke! LGTM
